app/league/[leagueId]/entry/[entryId]/week/WeekHelper.tsx

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { createWeeklyPicks } from '@/api/apiFunctions';
+import { parseUserPick } from '@/utils/utils';
+import Alert from '@/components/AlertNotification/AlertNotification';
+import { AlertVariants } from '@/components/AlertNotification/Alerts.enum';
+import { toast } from 'react-hot-toast';
+import { INFLTeam } from '@/api/apiFunctions.interface';
+import { IUser } from '@/api/apiFunctions.interface';
+import { IWeeklyPicks } from '@/api/apiFunctions.interface';
+
+interface IData {
+  target: {
+    value: string;
+  };
+}
+
+/**
+ * Handles the form submission.
+ * @param props - data, NFLTeams, user, entry, weeklyPicks, league, week, updateWeeklyPicks, setUserPick
+ * @param props.data - The form data.
+ * @param props.NFLTeams - Props for NFL teams
+ * @param props.user - Props for user
+ * @param props.entry - Prop for the entry string
+ * @param props.weeklyPicks - Props for the weeklyPicks
+ * @param props.league - Prop value for the leagueId in createWeeklyPicks
+ * @param props.week - Prop value for gameWeekId in updateWeeklyPicks
+ * @param props.updateWeeklyPicks - Prop for the updateWeeklyPicks function
+ * @param props.setUserPick - Prop for the setUserPick function
+ * @returns {void}
+ */
+export const onWeeklyPickChange = async ({
+  data,
+  entry,
+  league,
+  NFLTeams,
+  setUserPick,
+  updateWeeklyPicks,
+  user,
+  weeklyPicks,
+  week,
+}: {
+  data: IData;
```

Comment:
Move this entire object into an interface. Move them to an external Interface file.

Reply:
to clarify on this. do you just mean to move interface IData at the top of the file into the inface file and import it?